src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
```

Comment:
Can we use something more general for this state name? Like "warn"? 

Unless I'm misunderstanding, this _could_ be a component used for something else, right?